build.yaml

```diff
@@ -1584,6 +1584,17 @@ steps:
            --verbose \
            site/www/* \
            /io/www
+
+     # Grab the latest release of the docs
+     LATEST_02_SHA=$(gsutil cat gs://hail-common/builds/0.2/latest-hash/cloudtools-5-spark-2.4.0.txt 2>/dev/null)
+     gsutil cat gs://hail-common/builds/0.2/docs/hail-0.2-docs-$LATEST_02_SHA.tar.gz |
+         tar zxvf - -C /io/www --strip-components=1
```

Comment:
You're not handling the batch docs I don't think.

Reply:
Huh so I thought I was handling them correctly but agree it is not at all obvious, and took some playing with. Here's the layout:
- the `0.2` tarball expands into the `docs` directory which contains `0.2` and `batch`
- since `0.1` doesn't have batch, the tarball is structured so that it expands just to a `0.1` directory, so I expand that into the `www/docs` directory
![Screen Shot 2021-01-27 at 4 52 30 PM](https://user-images.githubusercontent.com/24440116/106059228-6934fe00-60c0-11eb-946c-d8ee566d48df.png)

Even so I am getting a 404 trying to looks at the batch docs on my dev deploy... I'll look into that

Reply:
Aha. So I suspect this is the problem:
https://github.com/hail-is/hail/blob/b125f2b46bf10c79bc7b13648976a0fb08249926/site/hail.nginx.conf.in#L36-L40
We don't create namespace prefix directories in `www`. The only reason this doesn't fail in prod is that `prefix` is "". I checked and dev deploy'ing main has this same problem.

I could remove the `@prefix@` or just remove the rule entirely and use the same redirects as we use for the `0.2` docs. Is there any reason to keep references to `pipeline` around?

Reply:
Confirmed that the batch docs load correctly after removing the `@prefix@` in the nginx config. The above question remains on whether we need the pipeline mentions anymore but up to you whether it's worth talking about in this PR.

Reply:
keep the pipeline mentions for now.

Reply:
Yeah this looks totally broken, I don't understand why I included `@prefix@` in the try_files. Nice catch y'all.